### ISSUE TITLE
Custom Content Files not loading?

### ISSUE THREAD

**Issue by werecrow** [larrychristensen/orcpub#153]
I've added a lot of content, custom races and sub races, archetypes, classes, etc

I still have those files on my computer, but they're not longer appearing, so like my rogue is no longer an arcane trickster, my (long story) elf sub race of cursed dragon winged tiger, no longer appears. I can readd this all of course, but a lot of work and hope I can remember it all

So asking, is there any way to make it see these files again, or should I just re-add it all?

**Comment by codeGlaze** [comment]
TLDR; Just re-uploaded your orcbrew files, that'll fill everything back in.

---

All custom content is stores in your browser's memory. Which means, eventually, data will be "lost" as other tabs or websites ask for the same storage space.

That's why you need to make sure to back up any custom content you make ASAP after making it. Then *verify it exported properly*.

There are random times when saving your content will make an "empty", or small sized, file. That can be fixed by just exporting the data again.

Hoping *one of these days* (tm) to help implement cloud storage so we don't need to worry so much about this.

**Comment by werecrow** [comment]
body{font-family: Geneva,Arial,Helvetica,sans-serif;font-size:9pt;background-color: #ffffff;color: black;}



Thank you, yeah, I was able to re-port the data and everything reappeared
And thank you for such a great product, sorry you had to remove all the content that was in all the base books, but at least it's not that hard to add it back inHave a nice day, thanks again
-----Original Message----- From: codeGlaze Sent: Oct 16, 2018 8:31 AM To: larrychristensen/orcpub Cc: werecrow , Author Subject: Re: [larrychristensen/orcpub] Custom Content Files not loading? (#153) 
TLDR; Just re-uploaded your orcbrew files, that'll fill everything back in.


All custom content is stores in your browser's memory. Which means, eventually, data will be "lost" as other tabs or websites ask for the same storage space.
That's why you need to make sure to back up any custom content you make ASAP after making it. Then verify it exported properly.
There are random times when saving your content will make an "empty", or small sized, file. That can be fixed by just exporting the data again.
Hoping one of these days (tm) to hemp implement cloud storage so we don't need to worry so much about this.

**Comment by Lukeybaby86** [comment]
i have had the same issue, but when i've tried to re-add it it's saying the files are not .orcbrew files, even though they are?
